src/test/features/claimant-response/routes/paid-in-full.ts

```diff
@@ -0,0 +1,72 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+
+import { checkAuthorizationGuards } from 'test/features/claimant-response/routes/checks/authorization-check'
+import { checkNotClaimantInCaseGuard } from 'test/features/claimant-response/routes/checks/not-claimant-in-case-check'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+
+import { Paths } from 'paid-in-full/paths'
+
+import { app } from 'main/app'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
```

Comment:
Please avoid relative paths. This one is `'test/http-mocks/draft-store'`

Reply:
Will make this change on PR #1219 👍Cheers